packages/ibm-products-styles/src/components/TagSet/_tag-set.scss

```diff
@@ -124,6 +124,8 @@ $block-class-modal: #{$_block-class}-modal;
   }
 
   .#{$block-class-overflow}__popover-trigger {
+    /* stylelint-disable-next-line declaration-no-important */
+    border: none !important;
```

Comment:
weird. i don't see this change showing up in the deploy preview. what is this doing? just curious because of the `important` usage.

Reply:
hey @davidmenendez, 
The `OperationalTag` from Carbon has a default border, so I have added this override to match our design.